fpfilter.pl

```diff
@@ -6,7 +6,8 @@
 use Getopt::Long;
 use IO::File;
 use File::Temp qw( tempdir );
-use File::Spec;
+use File::Spec qw( catfile );
```

Comment:
I don't think we need to include this `qw( catfile )` here.  We're always calling it explicitly as `File::Spec->catfile`.